frontend/src/routes.jsx

```diff
@@ -0,0 +1,153 @@
+// Constants
+import { appRoutes } from '@/constants/routes'
+// React components
+import ApiDocs from '@/components/ApiDocs'
+import Login from '@/layouts/authentication/components/Login'
+import OrganizationLayout from '@/layouts/organization/OrganizationLayout'
+import { ViewUsers } from '@/views/viewUsers'
+import { EditUser } from '@/views/editUser'
+import Layout from '@/layouts/Layout'
+import AdminUsersLayout from '@/layouts/admin/AdminUsersLayout'
+import PublicLayout from '@/layouts/PublicLayout'
+import ContactUs from '@/components/ContactUs'
+import Dashboard from '@/views/dashboard'
+
+// TODO: error bound component needs to be created
+export const routes = [
+  {
+    name: 'Login',
+    key: 'login',
+    path: appRoutes.login.main,
+    element: <Login />,
+    handle: { crumb: () => 'Login' }
+  },
+  // Contact-Us page
+  {
+    path: appRoutes.contactUs.main,
+    element: <PublicLayout crumbs />,
+    children: [
+      {
+        path: '',
+        element: <ContactUs />
+      }
+    ]
+  },
+  // Main application pages
+  {
+    path: '/',
+    element: <Layout crumbs />,
+    children: [
+      {
+        path: '',
+        element: <Navigate to={appRoutes.dashboard.main} replace />
+      },
+      // Docs
+      {
+        path: appRoutes.docs.main,
+        element: <ApiDocs />,
+        handle: { crumb: () => 'API Docs' }
+      },
+      // Dashboard
+      {
+        path: appRoutes.dashboard.main,
+        handle: { crumb: () => 'Dashboard' },
+        children: [
+          {
+            path: '',
+            element: <Dashboard />
+          }
+        ]
+      },
+      // Document
+      {
+        path: appRoutes.document.main,
+        handle: { crumb: () => 'Document' },
+        children: [
+          {
+            path: '',
+            element: (
+              <BCTypography variant="h4" sx={{ textAlign: 'center' }}>
+                Welcome to the Document!
+              </BCTypography>
+            )
+          }
+        ]
+      },
+      // Transactions
+      {
+        path: appRoutes.transactions.main,
+        handle: { crumb: () => 'Transactions' },
+        children: [
+          {
+            path: '',
+            element: (
+              <BCTypography variant="h4" sx={{ textAlign: 'center' }}>
+                Welcome to the Transactions!
+              </BCTypography>
+            )
+          }
+        ]
+      },
+      // Compliance Report
+      {
+        path: appRoutes.complianceReport.main,
+        handle: { crumb: () => 'Compliance Report' },
+        children: [
+          {
+            path: '',
+            element: (
+              <BCTypography variant="h4" sx={{ textAlign: 'center' }}>
+                Welcome to the Compliance Report!
+              </BCTypography>
+            )
+          }
+        ]
+      },
+      // Organization pages for IDIR User
+      {
+        path: appRoutes.organization.main,
+        handle: { crumb: () => 'Organization' },
```

Comment:
we can remove this line as we dont need breadcrumbs on that page according to the mockup

Reply:
Need more info on this, we can discuss this more and change accordingly.